src/lib.rs

```diff
@@ -53,26 +53,29 @@
 //! and run the main event loop:
 //!
 //! ```no_run
-//! # extern crate gtk;
-//! # use gtk::prelude::*;
-//! # use gtk::{Window, WindowType};
+//! extern crate gtk;
+//! extern crate gio;
+//!
+//! // To import all needed traits.
+//! use gtk::prelude::*;
+//! use gio::prelude::*;
+//!
+//! use std::env;
+//!
 //! fn main() {
-//!     gtk::init().unwrap();
-//!     // Create the main window.
-//!     let window = Window::new(WindowType::Toplevel);
-//!     // UI initialization.
-//!     // ...
-//!     // Don't forget to make all widgets visible.
-//!     window.show_all();
-//!     // Handle closing of the window.
-//!     window.connect_delete_event(|_, _| {
```

Comment:
All your examples contains
```
    window.connect_delete_event(|win, _| {
        win.destroy();
        Inhibit(false)
    });
```

Reply:
so IMHO it still needed here too

Reply:
Yes sir!

Reply:
@GuillaumeGomez, @sdroege Seems main windows destroyed normally even without `connect_delete_event`,
any reason to use it?